src/codegen/llvm/bindings.zig

```diff
@@ -53,10 +53,10 @@ pub const Module = opaque {
     extern fn LLVMDisposeModule(*Module) void;
 
     pub const setModulePICLevel = ZigLLVMSetModulePICLevel;
-    extern fn ZigLLVMSetModulePICLevel(module: *Module) void;
+    extern fn ZigLLVMSetModulePICLevel(module: *Module, big: bool) void;
 
     pub const setModulePIELevel = ZigLLVMSetModulePIELevel;
-    extern fn ZigLLVMSetModulePIELevel(module: *Module) void;
+    extern fn ZigLLVMSetModulePIELevel(module: *Module, large: bool) void;
 
     pub const setModuleCodeModel = ZigLLVMSetModuleCodeModel;
     extern fn ZigLLVMSetModuleCodeModel(module: *Module, code_model: CodeModel) void;
```

Comment:
This API usage should not exist, we already have the ability to modify the module freely.

Reply:
As I understand it, similar to float ABI, you can't set a code model on a bitcode module. So for these, we're stuck with the API (or `llc` later).

PIC/PIE are just module flags though, so I can look into emitting the module flags on our side in a follow-up PR if that's ok.

Reply:
> As I understand it, similar to float ABI, you can't set a code model on a bitcode module.

Correct for float ABI, but not for code model:
```llvm
!7 = !{i32 1, !"Code Model", i32 1}
```

Reply:
Filed https://github.com/ziglang/zig/issues/21238 so this isn't forgotten.